packages/components/src/tooltip/types.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import type * as Ariakit from '@ariakit/react/tooltip';
+/**
+ * Internal dependencies
+ */
+import type { PopoverProps } from '../popover/types';
+import type { ShortcutProps } from '../shortcut/types';
+
+export type TooltipProps = {
+	/**
+	 * The anchor for the tooltip.
+	 *
+	 * **Note**: Accepts only one child element.
+	 */
+	children: Ariakit.TooltipProps[ 'render' ];
```

Comment:
The `render` prop from `Ariakit` equals to

```ts
RenderProp | React.ReactElement
```

A couple of questions:

1. Do we intend to support the render prop case too in our `Tooltip`?
2. In any case, we'd need to update the README, since it currently says that `children` has type `Element`

Reply:
Thanks for pointing this out! I don't think it makes sense to support it, especially now, with our intention to match legacy. I think this resulted from my novice-level understanding of Typescript, but it is clearer to me now. I've updated this to `React.ReactElement` and the README to match. 🙂